lib/cc/engine/bundler_audit.rb

```diff
@@ -12,3 +13,16 @@ module BundlerAudit
     end
   end
 end
+
+# Patch Bundler::Audit::Scanner to prevent network access during insecure
+# source checks
+
+module Bundler
+  module Audit
+    class Scanner
+      def internal_host?(_uri)
+        false
```

Comment:
Unpatched, the scanner will ignore an insecure source if the host is internal, which is impossible to determine without network access. The `false` return value skips over the dns/network check, forcing the scanner to include every insecure source as a vulnerability.